packages/playground/windows/playground/App.cpp

```diff
@@ -0,0 +1,275 @@
+        L"       <ColumnDefinition Width='auto'/>"
+        L"     </Grid.ColumnDefinitions>"
+        L""
+        L"     <TextBlock"
+        L"       Grid.Column='0'"
+        L"       Margin='2,2,2,2'"
+        L"       VerticalAlignment='Center'"
+        L"       Text='JavaScript File (no extension):'/>"
+        L"     <ComboBox x:Name='x_JavaScriptFilename'"
+        L"       Grid.Column='1'"
+        L"       Margin='2,2,2,2'"
+        L"       MinWidth='250'"
+        L"       SelectedIndex='0'"
+        L"       VerticalAlignment='Center'"
+        L"       HorizontalAlignment='Stretch'>"
+        L"       <ComboBox.Items>"
+        L"         <ComboBoxItem Content='Samples\\rntester' />"
+        L"         <ComboBoxItem Content='Samples\\accessible' />"
+        L"         <ComboBoxItem Content='Samples\\animation' />"
+        L"         <ComboBoxItem Content='Samples\\calculator' />"
+        L"         <ComboBoxItem Content='Samples\\callbackTest' />"
+        L"         <ComboBoxItem Content='Samples\\click' />"
+        L"         <ComboBoxItem Content='Samples\\control' />"
+        L"         <ComboBoxItem Content='Samples\\flexbox' />"
+        L"         <ComboBoxItem Content='Samples\\focusTest' />"
+        L"         <ComboBoxItem Content='Samples\\geosample' />"
+        L"         <ComboBoxItem Content='Samples\\image' />"
+        L"         <ComboBoxItem Content='Samples\\index' />"
+        L"         <ComboBoxItem Content='Samples\\messages' />"
+        L"         <ComboBoxItem Content='Samples\\mouse' />"
+        L"         <ComboBoxItem Content='Samples\\scrollViewSnapSample' />"
+        L"         <ComboBoxItem Content='Samples\\simple' />"
+        L"         <ComboBoxItem Content='Samples\\text' />"
+        L"         <ComboBoxItem Content='Samples\\textinput' />"
+        L"         <ComboBoxItem Content='Samples\\ticTacToe' />"
+        L"         <ComboBoxItem Content='Samples\\view' />"
+        L"       </ComboBox.Items>"
+        L"     </ComboBox>"
+        L""
+        L"     <TextBlock"
+        L"       Grid.Column='2'"
+        L"       Margin='2,2,2,2'"
+        L"       VerticalAlignment='Center'"
+        L"       Text='App Name:'/>"
+        L"     <ComboBox"
+        L"       x:Name='x_ReactAppName'"
+        L"       Grid.Column='3'"
+        L"       Margin='2,2,2,2'"
+        L"       MinWidth='150'"
+        L"       SelectedIndex='0'"
+        L"       HorizontalAlignment='Stretch'"
+        L"       VerticalAlignment='Center' >"
+        L"       <ComboBox.Items>"
+        L"         <ComboBoxItem Content='RNTesterApp' />"
+        L"       </ComboBox.Items>"
+        L"     </ComboBox>"
+        L"   </Grid>"
+        L""
+        L"   <StackPanel"
+        L"     Grid.Row='1'"
+        L"     Grid.Column='1'"
+        L"     Orientation='Horizontal'>"
+        L"     <CheckBox x:Name='x_UseWebDebuggerCheckBox'"
+        L"       Margin='5,2'"
```

Comment:
Aren't these two mutually exclusive? If they are then this probably works better as a radio button group

Reply:
I'd rather leave this as is, so we can test what the host object does if the user sets these settings.